x-pack/test/security_solution_cypress/cypress/e2e/explore/host_details/risk_tab.cy.ts

```diff
@@ -10,43 +10,88 @@ import { visitHostDetailsPage } from '../../../tasks/navigation';
 
 import { cleanKibana, waitForTableToLoad } from '../../../tasks/common';
 import { TABLE_CELL, TABLE_ROWS } from '../../../screens/alerts_details';
+import { deleteRiskEngineConfiguration } from '../../../tasks/api_calls/risk_engine';
+import { openRiskInformationFlyout, enableRiskEngine } from '../../../tasks/entity_analytics';
+import { ALERTS_COUNT, ALERT_GRID_CELL } from '../../../screens/alerts';
+import { RISK_INFORMATION_FLYOUT_HEADER } from '../../../screens/entity_analytics';
+import { navigateToHostRiskDetailTab } from '../../../tasks/host_risk';
 
 describe('risk tab', { tags: ['@ess', '@serverless', '@brokenInServerless'] }, () => {
-  before(() => {
-    cleanKibana();
-    // illegal_argument_exception: unknown setting [index.lifecycle.rollover_alias]
-    cy.task('esArchiverLoad', { archiveName: 'risk_hosts' });
-  });
+  describe('with legacy risk score', () => {
+    before(() => {
+      cleanKibana();
```

Comment:
As the test is deleting the data that it matters, we might consider deleting the cleanKibana method.